src/python/install.sh

```diff
@@ -299,6 +299,9 @@ install_using_oryx() {
         echo "(!) Python version ${VERSION} already exists."
         exit 1
     fi
+
+    # The python install root path may not exist, so create it
+    mkdir -p "${PYTHON_INSTALL_PATH}"
```

Comment:
Can we bump the patch version in here as we are adding a new `option`?https://github.com/devcontainers/features/blob/cc560e605dbe2f631a66ec9463e19acb728db4e8/src/python/devcontainer-feature.json#L3

The Python Feature will automatically get published once this PR is merged. (Provided the version has changed)